### PR TITLE
feat(university-gateway): add RU xroad

### PR DESCRIPTION
**Pull request by jonarnarbriem**
# ...

Attach a link to issue if relevant

## What

Adding xroad config and connectors for RU

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/clients/university-application/reykjavik-university/src/lib/reykjavikUniversityClient.config.ts [hunk 1]
```diff
@@ -2,21 +2,27 @@ import { defineConfig } from '@island.is/nest/config'
...
-      },
+      xroadPath: env.required(
+        'XROAD_UNIVERSITY_GATEWAY_REYKJAVIK_UNIVERSITY_PATH',
+        'IS-DEV/EDU/10062/RvkUni-Hvin-Protected/umsoknir-v1',
+      ),
+      scope: [],
+      fetchTimeout:
+        env.optionalJSON(
+          'XROAD_UNIVERSITY_GATEWAY_REYKJAVIK_UNIVERSITY_TIMEOUT',
+        ) ?? 120000,
```
Comment (brynjarorng) [inline libs/clients/university-application/reykjavik-university/src/lib/reykjavikUniversityClient.config.ts:25]: This is a very long default timeout, this should be set to a lower value such as 10000

**Review by brynjarorng — CHANGES_REQUESTED** [review]
One minor change, otherwise LGTM

**Review by davidkef — APPROVED** [review]
lgtm

**Review by brynjarorng — APPROVED** [review]
LGTM

**Pull request merged**